core/src/main/kotlin/org/evomaster/core/problem/externalservice/HostnameResolutionAction.kt

```diff
@@ -4,18 +4,23 @@ import org.evomaster.core.search.EnvironmentAction
 import org.evomaster.core.search.StructuralElement
 import org.evomaster.core.search.gene.Gene
 
+class HostnameResolutionAction(
+    private val hostname: String,
+    val resolved: Boolean,
```

Comment:
what would be the point of having an action for a non-resolved hostname?
aren't we assuming that each hostname is non-resolved by default, unless explicitely stated?

if so, 2 points:
1) i guess here instead of a boolean `resolved` would need a resolved IP string
2) how are we going to deal with it in the generated tests were instrumentation is not applied?